inference/core/interfaces/stream/stream.py

```diff
@@ -199,9 +200,12 @@ def preprocess_thread(self):
                 if webcam_stream.stopped is True or self.stop:
                     break
                 else:
-                    self.frame, self.frame_cv, frame_id = webcam_stream.read()
+                    self.frame_cv, frame_id = webcam_stream.read_opencv()
                     if frame_id != self.frame_id:
                         self.frame_id = frame_id
+                        self.frame = Image.fromarray(
+                            cv2.cvtColor(self.frame_cv, cv2.COLOR_BGR2RGB)
+                        )
```

Comment:
`self.model.preprocess()` on line 209 can accept a numpy array, and in fact, when given a PIL image, it will convert it to a numpy array immediately. So here, no need to make `self.frame` a PIL image.
```suggestion
                        self.frame = cv2.cvtColor(self.frame_cv, cv2.COLOR_BGR2RGB)
```